.github/workflows/playwright email regression.yml

```diff
@@ -0,0 +1,39 @@
+name: Playwright Email Tests
+on:
+  workflow_dispatch:
+ 
```

Comment:
Yes, I didn't want it spamming every time we push stuff into the repo at the moment because it will re-trigger on every push. I could add in a time based cron for the time being so we are making sure it's running at least once a day like the Postman ones do.